broker-daemon/block-order-worker/index.js

```diff
@@ -61,6 +63,10 @@ class BlockOrderWorker extends EventEmitter {
     const filterOrdersWithOrderId = (key, value) => !!Order.fromStorage(key, value).orderId
     const getOrderIdFromOrder = (key, value) => Order.fromStorage(key, value).orderId
 
+    // create a subset of active orders and fills
```

Comment:
[nit] maybe to be more clear it could say `create a subset containing active orders and fills`